rdpdr.c

```diff
@@ -680,11 +724,15 @@ rdpdr_process_irp(STREAM s)
 				break;
 			}
 
+			/* DR_CONTROL_REQ (2.2.1.4.5 of MS-RDPEFS) */
+			/* OutputBufferLength */
 			in_uint32_le(s, bytes_out);
 			in_uint32_le(s, bytes_in);
 			in_uint32_le(s, request);
+			/* Padding */
 			in_uint8s(s, 0x14);
 
+			/* TODO: Why do we need to increase lenth by padlen? Or is it hdr len? */
```

Comment:
I guess "lenth" should be "length".

Reply:
Yep. You're right. Thank you for catching this up:)